### PR TITLE
replace 3rdparty transcript urls

### PR DESCRIPTION
**Pull request by muhammad-ammar**
[EDUCATOR-1278](https://openedx.atlassian.net/browse/EDUCATOR-1278)

### CODE REVIEW

File: instance_config.yaml [hunk 1]
```diff
@@ -55,6 +55,13 @@ s3_base_url: https://s3.amazonaws.com
 aws_video_transcripts_bucket:
 aws_video_transcripts_prefix: video-transcripts/
 
+# cielo24 api urls
+cielo24_api_base_url: https://api.cielo24.com/api
```
Comment (muzaffaryousaf) [inline instance_config.yaml]: Can we use sandbox here by default ?

**Comment by muzaffaryousaf** [comment]
@Qubad786 can you give it a quick look too.

File: control/tests/test_deliver_cielo.py [hunk 2]
```diff
@@ -46,6 +46,7 @@ def setUp(self):
             'preferred_languages': ['en', 'ur'],
             's3_video_url': 'https://s3.amazonaws.com/bkt/video.mp4',
             'callback_base_url': 'https://veda.edx.org/cielo24/transcript_completed/1234567890',
+            'cielo24_api_base_url': 'https://api.cielo24.com/api',
```
Comment (muzaffaryousaf) [inline control/tests/test_deliver_cielo.py]: let's use sandbox url in tests.

File: instance_config.yaml [hunk 3]
```diff
@@ -55,6 +55,13 @@ s3_base_url: https://s3.amazonaws.com
...
+
+# 3playmedia api urls
+three_play_api_base_url: https://api.3playmedia.com/
+three_play_api_transcript_url: https://static.3playmedia.com/
+
```
Comment (muzaffaryousaf) [inline instance_config.yaml]: We should add these variables to the test_config file as well.

**Comment by muhammad-ammar** [comment]
@muzaffaryousaf addressed your feedback.

File: VEDA_OS01/transcripts.py [hunk 4]
```diff
@@ -36,20 +36,39 @@
...
+    CONFIG['three_play_api_base_url'],
+    'files/{file_id}/translations/order'
+)
+THREE_PLAY_TRANSLATION_STATUS_URL = utils.build_url(
+    CONFIG['three_play_api_transcript_url'],
+    'files/{file_id}/translations/{translation_id}'
+)
+THREE_PLAY_TRANSLATION_DOWNLOAD_URL = utils.build_url(
+    CONFIG['three_play_api_transcript_url'],
+    'files/{file_id}/translations/{translation_id}captions.srt'
```
Comment (Qubad786) [inline VEDA_OS01/transcripts.py]: `files/{file_id}/translations/{translation_id}/captions.srt`

File: control/tests/test_deliver_cielo.py [hunk 5]
```diff
@@ -46,6 +46,7 @@ def setUp(self):
             'preferred_languages': ['en', 'ur'],
             's3_video_url': 'https://s3.amazonaws.com/bkt/video.mp4',
             'callback_base_url': 'https://veda.edx.org/cielo24/transcript_completed/1234567890',
+            'cielo24_api_base_url': 'https://sandbox.cielo24.com/api',
```
Comment (Qubad786) [inline control/tests/test_deliver_cielo.py]: are we missing `/` in the end?

Comment (muhammad-ammar) [reply control/tests/test_deliver_cielo.py]: nope

File: VEDA_OS01/transcripts.py (hunk 4 above)
Comment (muhammad-ammar) [reply VEDA_OS01/transcripts.py]: good catch

**Comment by muzaffaryousaf** [comment]
Don't forget to squash before merging.

**Pull request merged**